plugins/azure/resoto_plugin_azure/resource/compute.py

```diff
@@ -2499,6 +2530,9 @@ class AzureVirtualMachine(AzureResource):
         "user_data": S("properties", "userData"),
         "virtual_machine_scale_set": S("properties", "virtualMachineScaleSet", "id"),
         "vm_id": S("properties", "vmId"),
+        "instance_cores": S("properties", "hardwareProfile", "vmSizeProperties", "vCPUsAvailable", default=0.0)
+        >> F(lambda x: float(x)),
+        "instance_type": S("type"),
```

Comment:
does this property even exists?

It should be taken from hardwareProfile.vmSize.